squad/core/models.py

```diff
@@ -825,10 +827,40 @@ def __init__(self, build):
 
 class Subscription(models.Model):
     project = models.ForeignKey(Project, related_name='subscriptions')
-    email = models.CharField(max_length=1024, validators=[EmailValidator()])
+    email = models.CharField(
+        max_length=1024,
+        null=True,
+        blank=True,
+        validators=[EmailValidator()]
+    )
+    user = models.ForeignKey(
```

Comment:
this field is empty by default. I think we need to fill it in with existing subscriptions in a migration. I checked a few users and none of them had any 'new' subscriptions. There were however 'old' ones with corresponding emails.